udata_hydra/datalake_service.py

```diff
@@ -115,14 +126,21 @@ async def process_resource(url: str, dataset_id: str, resource_id: str, response
         document = {
             'analysis:error': None,
             'analysis:filesize': filesize,
-            'analysis:mime': mime_type
+            'analysis:mime': mime_type,
         }
+        # Check if checksum has been modified
+        # TODO: improve file modification logic
+        checksum_modified = await has_checksum_been_modified(resource_id, sha1)
+        if checksum_modified:
+            document['analysis:checksum_last_modified'] = datetime.now().isoformat()
```

Comment:
Shouldn't we make sure we're using UTC? `utcnow()`

Reply:
Done :+1: